crates/core_arch/src/x86/avx512f.rs

```diff
@@ -8852,6 +8852,36 @@ pub unsafe fn _mm512_maskz_expand_epi32(k: __mmask16, a: __m512i) -> __m512i {
     ))
 }
 
+/// Load contiguous active 32-bit integers from unaligned memory at mem_addr (those with their respective bit set in mask k), and store the results in dst using writemask k (elements are copied from src when the corresponding mask bit is not set).
+///
+/// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=512_mask_expandloadu_epi32&expand=2352)
+#[inline]
+#[target_feature(enable = "avx512f")]
+#[cfg_attr(test, assert_instr(vpexpandd))]
+pub unsafe fn _mm512_mask_expandloadu_epi32(
+    src: __m512i,
+    k: __mmask16,
+    mem_addr: *const i32,
+) -> __m512i {
+    let load = ptr::read_unaligned(mem_addr as *const __m512i);
```

Comment:
Same here.

Reply:
You also need to remove `expandloadu` since it reads more than it should, which could cause out-of-bounds reads.